index.html

```diff
@@ -303,6 +301,50 @@ <h5 class="header-5">About Me</h5>
             </li>
         </ul>
     </section>
-</body>
-
+    <section id="contact-form">
+        <div id="header-6">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h3>
+        </div>
+        <div>
+            <form action="https://formspree.io/f/xvolakyp" method="POST">
+                <input class="form-input" type="text" name="fullname" placeholder="Full name" required maxlength="30" />
+                <input class="form-input" type="email" name="email" placeholder="Email address" required />
+                <textarea id="text-area" name="message" required maxlength="500">
+Enter text here</textarea
+          >
+          <button class="button-3h" type="submit">Get in touch</button>
+        </form>
```

Comment:
- [ ] Good job on adding button but please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) and kindly add button interactions as it is shown on design:

![image](https://user-images.githubusercontent.com/74506933/154165875-b279512d-2a91-49e5-ac5b-cb02623103e3.png)